### PR TITLE
Fix usage of trip update delay

### PR DESCRIPTION
**Pull request by mjaakko**
Fix for:
image

### CODE REVIEW

**Review by kenkus-futurice — APPROVED** [review]
LGTM

File: src/main/java/fi/hsl/transitdata/railsource/RailSpecific.java [hunk 1]
```diff
@@ -14,4 +14,21 @@
                 .filter(GtfsRealtime.FeedEntity::hasTripUpdate)
                 .collect(Collectors.toList());
     }
+
+    static GtfsRealtime.TripUpdate fixInvalidTripUpdateDelayUsage(GtfsRealtime.TripUpdate tripUpdate) {
+        //If trip update has specified delay and stop time updates, timing information in stop time updates is ignored
+        //-> remove delay from trip update
+        if (tripUpdate.hasDelay() && tripUpdate.getStopTimeUpdateCount() != 0) {
+            GtfsRealtime.TripUpdate fixedTripUpdate = GtfsRealtime.TripUpdate.newBuilder()
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/transitdata/railsource/RailSpecific.java]: Maybe possible to do something like?:
`tripUpdate.toBuilder().setDelay(null).build()`
I don't know. Do you know?

**Review by kenkus-futurice — APPROVED** [review]
LGTM

File: src/main/java/fi/hsl/transitdata/railsource/RailSpecific.java (hunk 1 above)
Comment (mjaakko) [reply src/main/java/fi/hsl/transitdata/railsource/RailSpecific.java]: Seems to be possible to use `clearDelay()` to remove delay field. Fixed.

**Review by kenkus-futurice — APPROVED** [review]
LGTM

**Pull request merged**